ibis/bigquery/tests/conftest.py

```diff
@@ -0,0 +1,80 @@
+import uuid
+import pytest
+
+import ibis
+import pandas as pd
+bq = pytest.importorskip('google.datalab.bigquery')
+from ibis.bigquery import client as ibc
+
+
+PROJECT_ID = 'bq-for-ibis'
+DATASET_ID = 'ci_{}'.format(uuid.uuid4()).replace('-', '_')
+TABLE = 'table_{}'.format(uuid.uuid4()).replace('-', '_')
+
+
+
+@pytest.fixture(scope='session')
+def bq_context():
+    return ibc._bq_get_context(PROJECT_ID)
+
+
+@pytest.fixture(scope='session')
+def bq_datasets(bq_context):
+    return bq.Datasets(bq_context)
+
+
+@pytest.fixture(scope='session')
+def bq_dataset(bq_context):
+    return bq.Dataset(DATASET_ID, context=bq_context)
+
+
+@pytest.fixture(scope='session')
+def bq_table(bq_dataset):
+    name = bq_dataset.name.dataset_id + '.' + TABLE
+    # can't use client.table: it needs to exist so we can get a schema
+    return bq.Table(name, context=bq_dataset._context)
+
+
+@pytest.fixture(scope='session')
+def df():
+    return pd.DataFrame(dict(
+        int_column=[1], float_column=[2.],
+        string_column=['a'], bool_column=[True],
+    ))
+
+
+@pytest.fixture(scope='session')
+def client(bq_dataset):
+    bq_name = bq_dataset.name
+    client = ibc.BigQueryClient(bq_name.project_id, bq_name.dataset_id)
+    ibis.options.default_backend = client
```

Comment:
Don't set this for unit tests unless you are explicitly testing this, because it modifies global state.